src/net/db.rs

```diff
@@ -161,11 +161,11 @@ impl LocalPeer {
 
         let addr = addrbytes;
         let port = port;
-        let services = ServiceFlags::RELAY;
+        let services = (ServiceFlags::RELAY as u16) | (ServiceFlags::RPC as u16);
 
         info!(
-            "Will be authenticating p2p messages with public key: {}",
-            Secp256k1PublicKey::from_private(&pkey).to_hex()
+            "Will be authenticating p2p messages with public key: {}, service: {}",
+            Secp256k1PublicKey::from_private(&pkey).to_hex(), &to_hex(&(services as u16).to_be_bytes());
```

Comment:
Use a structured log here (the log entry already is implicitly written as 2 keys and values).

Reply:
added